x-pack/plugins/siem/cypress/objects/rule.ts

```diff
@@ -4,7 +4,12 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-export const totalNumberOfPrebuiltRules = 127;
+// eslint-disable-next-line
```

Comment:
Is this a `no-restricted-paths` error? It's usually helpful to narrow down the `eslint-disable` as much as possible for both context and scope.